.github/workflows/fetch-observability-packs.yml

```diff
@@ -0,0 +1,100 @@
+name: Fetch Observability Packs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # midnight
+
+env:
+  BOT_NAME: nr-opensource-bot
+  BOT_EMAIL: opensource+bot@newrelic.com
+  NODE_OPTIONS: '--max-old-space-size=4096'
+
+jobs:
+  fetch-observability-packs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: main
+
+      - name: Setup node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Cache dependencies
+        id: yarn-cache
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Fetch observability packs
+        run: yarn run fetch-observability-packs
+        env:
+          NR_GQL_URL: 'https://api.newrelic.com/graphql'
+          NR_API_TOKEN: ${{ secrets.NR_API_TOKEN }}
+
+      - name: Temporarily disable branch protection
+        id: disable-branch-protection
+        uses: actions/github-script@v1
+        with:
+          github-token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+          previews: luke-cage-preview
+          script: |
+            const result = await github.repos.updateBranchProtection({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              branch: 'main',
+              required_status_checks: null,
+              restrictions: null,
+              enforce_admins: null,
+              required_pull_request_reviews: null
+            })
+            console.log("Result:", result)
+
+      - name: Commit changes
+        id: commit-changes
+        run: |
+          git config --local user.email "${{ env.BOT_EMAIL }}"
+          git config --local user.name "${{ env.BOT_NAME }}"
+          git add ./src/data/observability-packs.json
+          git diff-index --quiet HEAD ./src/data/observability-packs.json || git commit -m 'chore(observability-packs): updated observability packs'
+          echo "::set-output name=commit::true"
```

Comment:
If this step fails (perhaps because there isn't any new information to commit), will that cause the remaining steps to error out? Just want to double-check that we're not opening the door to disabling the branch protection and failing before we can reenable it.

Reply:
The jobs that reenable branch protection will always run, no matter what the outcome of this job is.

Reply:
It has the `if: always()` setting